includes/amp-helper-functions.php

```diff
@@ -985,19 +985,43 @@ function amp_get_schemaorg_metadata() {
 	 *
 	 * @param string $schema_img_url URL of the publisher logo, either the Custom Logo or the Site Icon.
 	 */
-	$filtered_schema_img_url = apply_filters( 'amp_site_icon_url', $schema_img['url'] );
-	if ( $filtered_schema_img_url !== $schema_img['url'] ) {
-		$schema_img['url'] = $filtered_schema_img_url;
-		unset( $schema_img['width'], $schema_img['height'] ); // Clear width/height since now unknown, and not required.
+	$filtered_schema_img_url = apply_filters( 'amp_site_icon_url', $logo_image_url );
+	if ( $filtered_schema_img_url !== $logo_image_url ) {
+		$logo_image_url = $filtered_schema_img_url;
```

Comment:
Yeah, good point. The only need for this  conditional was this line:

```php
unset( $schema_img['width'], $schema_img['height'] ); // Clear width/height since now unknown, and not required.
```

But it was removed.

Reply:
Yeah, it was useful 😄